src/main/java/dk/kb/present/View.java

```diff
@@ -118,4 +146,31 @@ public String toString() {
                ", transformers=" + super.toString() +
                ')';
     }
+
+    /**
+     * If record has children, the first child is returned.
+     * @param record to extract the newest child from.
+     * @return  the data from the first child related to the input record.
+     */
+    private String getFirstChild(DsRecordDto record) {
+        // TODO: Figure how to choose correct manifestation for record, if more than one is present
+        // Return first child record, but if there are multiple presentation manifestations,
+        // the rest are currently not added to the transformation
+        return record.getChildren() == null ? "" :
+                record.getChildren().stream()
+                        .map(this::getNonNullChild)
```

Comment:
Is this the proper logic for Preservica-records? I thought that the type of manifestation played a role!?

Reply:
It does play a role! However, I believe that is handled in this PR https://github.com/kb-dk/ds-present/pull/71, which we converted to a draft until this one is through?

Reply:
Ah! Of course. My bad.